preprocessing/find_media.js

```diff
@@ -55,6 +72,12 @@ function findValidMedia(filenames) {
   return null;
 }
 
+// Check if a video file's path ends in the ".youtube" extension. 
+function isVideoFilepathYoutubeExtension(videoFile) {
+  const lengthOfExtension = 8; // length of .youtube extension is 8
```

Comment:
This can be written more concisely as:
`return videoFile.endsWith(".youtube");

Reply:
(Come to think of it, a lot of other LingView code could be made cleaner by using endsWith, too. Thoughts for the future.)

Reply:
Updated.